.vscode/settings.json

```diff
@@ -8,7 +8,7 @@
         "**/logs/": true,
         "**/obj/": true,
         "**/packages/": true,
-        "tools/": true,
+        "/tools/": true,
```

Comment:
Would this make sense as **/tools/?

Reply:
No, since I think we only want to ignore the root tools directory. This was causing /src/tools to get ignored.